internal/start/start.go

```diff
@@ -98,6 +98,7 @@ type kongConfig struct {
 	PgmetaId    string
 	DenoRelayId string
 	LogflareId  string
+	ApiPort     uint
```

Comment:
This should really be `uint16` but we can probably punt on it to a later time.

Reply:
Yhea, the base type is currently `uint` 
https://github.com/supabase/cli/blob/main/internal/utils/config.go#L102